foundry/applications/formApplications/baseEntitySheets/journalSheet.d.ts

```diff
@@ -26,7 +26,7 @@ declare class JournalSheet<
   get title(): string;
 
   /** @override */
-  getData(options?: Application.RenderOptions): Promise<D> | D;
+  getData(options?: Application.RenderOptions): D;
```

Comment:
By doing this, you prevent people from creating custom `SheetClasses` that have `async` `getData` methods. Please revert this.